src/main/java/com/revature/domain/Subject.java

```diff
@@ -0,0 +1,76 @@
+package com.revature.domain;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name="ie_subject")
+public class Subject {
+	
+	@Id
+	@Column(name="s_id")
+	@GeneratedValue(strategy=GenerationType.SEQUENCE, generator="subjectSeq")
+	@SequenceGenerator(allocationSize=1, name="subjectSeq", sequenceName="subject_seq")
+	private Integer id;
+	
+	@Column(name = "s_subject")
+	private String subject;
+	
+	public Subject(){}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)